app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = current_user;
```

Comment:
Nothing wrong with this, except that it's unnecessary. Just use current_user in your view. It's available in controllers, views and helpers. No need to set up *another* variable. Most people do, until they get used to devise. It's a GREAT gem, just has some quirks. But that's because it does a lot.